load/listener.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
+import (
+	"context"
+	"sync"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/set"
+
+	"github.com/ava-labs/hypersdk/api/ws"
+	"github.com/ava-labs/hypersdk/chain"
+)
+
+var _ Listener[*chain.Transaction] = (*DefaultListener)(nil)
+
+type DefaultListener struct {
+	client  *ws.WebSocketClient
+	tracker Tracker[ids.ID]
+
+	lock          sync.RWMutex
+	issuedTxs     uint64
+	lastIssued    bool
+	receivedTxs   uint64
+	inFlightTxIDs set.Set[ids.ID]
+}
+
+// NewDefaultListener creates a new DefaultListener instance.
+// Set txTarget to 0 to never stop the [DefaultListener.Listen] method unless the context is cancelled.
```

Comment:
What is `txTarget` ?

Reply:
Good catch! Outdated comment, it's removed!